packages/blade/src/components/Box/BaseBox/storybookArgTypes.ts

```diff
@@ -0,0 +1,242 @@
+import { getOnlyBoxProps } from '../Box';
+import { filterStyledProps } from '../styled-props';
+import type { BaseBoxProps, BoxProps, StyledProps } from './types';
+
+type StorybookArgTypes<T> = {
+  [P in keyof T]: {
+    table?: { category?: 'StyledProps' | null; disable?: boolean };
+    control?: { type: string };
+    description?: string;
+  };
+};
+
+const commonProperties = {
+  control: {
+    type: 'object',
+  },
+};
+
+const spacingTypeShorthandDescription = [
+  'Can be',
+  '- an absolute value like `"10px"`',
+  '- token `"spacing.5"`',
+  '- an array shorthand `["spacing.2", "10px", "spacing.5", "spacing.9"]` (Similar to CSS shorthands)',
```

Comment:
```suggestion
  '- an array shorthand `["spacing.2", "10px", "spacing.5", "spacing.9"]` (Similar to CSS shorthand padding-top, padding-right, padding-bottom, padding-left)',
```

Reply:
I added link to MDN shorthands. This might get confusing and people would think we're setting `paddingTop` property. We are just using CSS shorthand instead that already exists.